src/documentation/index.js

```diff
@@ -0,0 +1,41 @@
+export const options = {
+    definition: {
+        openapi: '3.0.0',
+        info: {
+          title: 'E-commerce api',
+          description: 'E-commerce api docs ',
+          version: '0.7.5',
+        },
+
+        servers:[{
+            url:process.env.SERVER_URL
+        }],
+        schemes: ['HTTP', 'HTTPS'],
```

Comment:
this will no work

Reply:
It seems like JSDOC  format does not support schemes. I think it will require to write the documentation in another format maybe in JSON format.

Reply:
it's support it know it